_posts/2018/02/2018-02-16-library-carpentry-governance.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "Library Carpentry Governance - An Update"
+date: 2018-02-16
+time: "00:00:00"
+category: [ "Library Carpentry", "Governance", "Community Building"]
+---
+
+Eight of us from Australia, the UK, the US, Canada and South Africa met this week via zoom to discuss taking 
+the [Library Carpentry](https://github.com/LibraryCarpentry) project forward.
+
+We discussed the creation of a new website to replace [this one](http://librarycarpentry.github.io/), talked about 
+what constitutes a Library Carpentry workshop, discussed who can teach the material, and what kind of governance structure 
+we need to take the work forward. There was an update on the status of the [IMLS-funded Library Carpentry Coordinator role to be based at CDL](http://www.datacarpentry.org/jobs/).
+
+### The website
+
+[Richard Vankoningsveld](https://twitter.com/richyvk) has done a great draft of 
+a [new website](https://www.richyvk.me/library-carpentry-redesign/). 
+Can people please have a look at it and raise issues that need fixing on the repo itself? 
```

Comment:
We would be grateful for feedback - please have a look and file issues in its repository (ADD LINK TO REPO).